gossip/src/duplicate_shred.rs

```diff
@@ -30,7 +31,7 @@ pub struct DuplicateShred {
     pub(crate) wallclock: u64,
     pub(crate) slot: Slot,
     _unused: u32,
-    shred_type: ShredType,
+    _unused_shred_type: ShredType,
```

Comment:
now that we can have a conflict between data and coding shreds, this field is useless.